README.md

```diff
@@ -461,3 +461,14 @@ The `ignore` flag, when used with the `bootstrap` command, can also be set in `l
 
 > Hint: The glob is matched against the package name defined in `package.json`,
 > not the directory name the package lives in.
+
+#### --onlyExplicitUpdates
+
+Only will bump versions for packages that have been updated explicitly rather than cross-dependencies.
```

Comment:
Might be worth including a warning against using this with a major version bump.